__tests__/app.test.js

```diff
@@ -86,6 +86,7 @@ describe("/api/reviews/:review_id", () => {
                     created_at: expect.any(String),
                     votes: expect.any(Number),
                     review_body: expect.any(String),
+                    comment_count: expect.any(Number),
```

Comment:
For this test, we don't want to change our old test - we want to make a new test as otherwise, this would not be in the spirit of TDD!
We want to make sure our tests are buildable and we can easily extend our code.
If this test fails after you've added the new one, it was too brittle - since you are using .toMatchObject though, it should be fine